### ISSUE TITLE
Cancellation requires a Contact

### ISSUE THREAD

**Issue by SethPate** [Gig-o-Matic/GO3#295]
<img width="757" alt="image" src="https://github.com/Gig-o-Matic/GO3/assets/17327269/dad42e7b-a342-47ac-ad95-ed981b35dc81">

I'd like to cancel this gig, but I'm asked to list a contact before cancelling. On Mobile, the window doesn't refocus after clicking 'save', so it's hard to tell what the error is. On desktop, the refocus does work, but probably we should be able to cancel without listing a contact.

**Comment by aaronoppenheimer** [comment]
This is an artifact of the way gigs were imported from the old gig-o. New gigs are always created with a contact required; when we imported them the contact wasn't set and the gig-o handles it badly. Ben has changed the import process so that it now sets a contact, but for gigs imported before we figured this out, there's no contact. If you set one and then save the gig, it'll act like you expect. Sorry about that!
